models/User.go

```diff
@@ -1933,6 +1933,12 @@ func PerformOrderFillTransaction(ask *Ask, bid *Bid, stockTradePrice uint64, sto
 		bidStatus = BidUndone
 	}
 
+	// HACK. The line below is a hack to ensure that nothing breaks in existing code.
```

Comment:
This thing will most likely fail at the client. The client will deduct the stocks twice - once when PlaceOrderTx is recvd, and then when orderfilltx is recvd